docs-gen/config.toml

```diff
@@ -7,17 +7,17 @@ publishDir = "../docs"
   # Change default color scheme with a variant one. Can be "red", "blue", "green".
   themeVariant = "green"
   disableInlineCopyToClipBoard = true
-  editURL = "https://github.com/genivi/vehicle_signal_specification/edit/master/docs-gen/content/"
+  editURL = "https://github.com/covesa/vehicle_signal_specification/edit/master/docs-gen/content/"
```

Comment:
I guess it does not matter if we use "covesa" or "COVESA", but what is our preferred way of referring to the COVESA githubs site, lower case or upper case?

Reply:
I checked and it doesn't matter. Before it was mixed. I personally prefer `covesa` in URLs.

Reply:
Agree, looks nicer.